CMakeLists.txt

```diff
@@ -28,15 +31,8 @@ include_directories(${MSP_INCLUDE_DIR})
 ################################################################################
 ### libraries
 
-# low-level API
-add_library(msp ${MSP_SOURCE_DIR}/MSP.cpp)
```

Comment:
Update the README to reflect the change in the API, i.e. the removal of the `MSP` class.

Reply:
can do